### PR TITLE
Tajara Food Expansion

### PR DESCRIPTION
**Pull request by Spookat**
Adds new Tajara-themed foods to the game. Mini-avahs, earthen-root dishes, seafood expansion, bread pudding and more. 17 new dishes total.

All sprites done by AfraidofAmericans#8428.

### CODE REVIEW

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 1]
```diff
@@ -5336,6 +5397,14 @@
 	It is known for its hard crust, bland taste and for being long lasting. The hard bread was usually prepared for long journeys, hard winters or military campaigns, \
 	due to its shelf life. Certain folk stories and jokes claim that such food could also be used as an artillery ammunition or thrown at besieging armies during sieges."
 
+/obj/item/reagent_containers/food/snacks/hardbread_slice
+	name ="adhomian hard bread slice"
+	desc = "A long-lasting tajaran bread slice. It is usually prepared for long journeys, hard winters or military campaigns."
```
Comment (FluffyGhoster) [inline code/modules/reagents/reagent_containers/food/snacks.dm:5402]:
```suggestion
	desc = "A long-lasting Tajaran bread slice. It is usually prepared for long journeys, hard winters or military campaigns."
```

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 2]
```diff
@@ -5419,6 +5488,195 @@
...
+	desc = "TBD"
+	icon_state = "earthenroot_salad"
+	trash = /obj/item/trash/snack_bowl
+	reagents_to_add = list(/singleton/reagent/nutriment = 10)
+	reagent_data = list(/singleton/reagent/nutriment = list("sweet chunky earthen-root" = 2, "sour cream" = 2, "oily berries" = 2, "sweet herbs" = 2))
+	bitesize = 3
+
+/obj/item/reagent_containers/food/snacks/stuffed_earthenroot
+	name = "stuffed earthen-root"
+	desc = "TBD"
```
Comment (FluffyGhoster) [inline code/modules/reagents/reagent_containers/food/snacks.dm:5542]: Missing desc?

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 3]
```diff
@@ -5419,6 +5488,195 @@
...
+	icon_state = "earthenroot_wedges"
+	trash = /obj/item/trash/plate
+	filling_color = "#3884a7"
+	reagents_to_add = list(/singleton/reagent/nutriment = 4, /singleton/reagent/nutriment/triglyceride/oil = 1.2)
+	reagent_data = list(/singleton/reagent/nutriment = list("crispy sweet earthen-root wedges" = 4))
+	bitesize = 2
+
+/obj/item/reagent_containers/food/snacks/salad/earthenroot
+	name = "earthen-root salad"
+	desc = "TBD"
```
Comment (FluffyGhoster) [inline code/modules/reagents/reagent_containers/food/snacks.dm:5533]: Missing desc?

File: html/changelogs/ramke - tajfoodexpansion.yml [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - rscadd: "Tajara-themed food expansion, 17 new dishes."
```
Comment (FluffyGhoster) [inline html/changelogs/ramke - tajfoodexpansion.yml:41]: A list of dishes would be nice to have in the changelog

**Review by FluffyGhoster — CHANGES_REQUESTED** [review]

File: code/modules/cooking/recipes/cultural/recipes_tajara.dm [hunk 5]
```diff
@@ -187,3 +302,17 @@
 		/obj/item/reagent_containers/food/snacks/meat/adhomai
 	)
 	result = /obj/item/reagent_containers/food/snacks/explorer_ration
+
+// Tajaran dips
+
+/singleton/recipe/sarmikhir
```
Comment (SleepyGemmy) [inline code/modules/cooking/recipes/cultural/recipes_tajara.dm:308]:
```suggestion
// Tajaran Dips
/singleton/recipe/sarmikhir
```

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 6]
```diff
@@ -5336,6 +5397,14 @@
 	It is known for its hard crust, bland taste and for being long lasting. The hard bread was usually prepared for long journeys, hard winters or military campaigns, \
 	due to its shelf life. Certain folk stories and jokes claim that such food could also be used as an artillery ammunition or thrown at besieging armies during sieges."
 
+/obj/item/reagent_containers/food/snacks/hardbread_slice
+	name ="adhomian hard bread slice"
+	desc = "A long-lasting Tajaran bread slice. It is usually prepared for long journeys, hard winters or military campaigns."
```
Comment (SleepyGemmy) [inline code/modules/reagents/reagent_containers/food/snacks.dm:5402]:
```suggestion
	name = "adhomian hard bread slice"
	desc = "A long-lasting Tajaran bread slice. It is usually prepared for long journeys, hard winters, or military campaigns."
```

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 7]
```diff
@@ -5419,6 +5488,199 @@
...
+/obj/item/reagent_containers/food/snacks/tunneler_souffle
+	name = "tunneler souffle"
+	desc = "A souffle made from an Adhomian ice tunneler egg and traditional sour cream."
+	icon_state = "tunneler_souffle"
+	reagents_to_add = list(/singleton/reagent/nutriment/protein/egg = 3, /singleton/reagent/nutriment = 5, /singleton/reagent/drink/milk/cream = 5)
+	reagent_data = list(/singleton/reagent/nutriment = list("baked dough" = 2))
+
+/obj/item/reagent_containers/food/snacks/chipplate/miniavah_basket
+	name = "mini-avah basket"
+	desc = "A basket of mini-Avahs, a small variant of a traditional Avah filled with meat that are the perfect size for dipping."
```
Comment (SleepyGemmy) [inline code/modules/reagents/reagent_containers/food/snacks.dm:5614]:
```suggestion
	desc = "A basket of mini-avahs, a small variant of a traditional avah filled with meat that are the perfect size for dipping."
```

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 8]
```diff
@@ -5419,6 +5488,199 @@
...
+	bitesize = 2
+	filling_color = "#e4e4e4"
+
+/obj/item/reagent_containers/food/snacks/chip/miniavah/tajhummus
+	name = "hummus mini-avah"
+	desc = "A miniature Avah filled with meat. This one has sweet hummus on it."
+	icon_state = "avah_full_hummus"
+	bitten_state = "avah_half_hummus"
+	bitesize = 2
+	filling_color = "#cca628"
```
Comment (SleepyGemmy) [inline code/modules/reagents/reagent_containers/food/snacks.dm:5682]:
```suggestion
/obj/item/reagent_containers/food/snacks/chip/miniavah
	name = "mini-avah"
	desc = "A miniature avah, a traditional Adhomian treat. This one is filled with meat, and is the perfect size for dipping."
	icon_state = "avah_full"
	bitten_state = "avah_half"
	bitesize = 3

/obj/item/reagent_containers/food/snacks/chip/miniavah/cheese
	name = "cheese mini-avah"
	desc = "A miniature avah filled with meat. This one has cheese on it."
	icon_state = "avah_full_queso"
	bitten_state = "avah_half_queso"
	bitesize = 2
	filling_color = "#FFF454"

/obj/item/reagent_containers/food/snacks/chip/miniavah/salsa
	name = "salsa mini-avah"
	desc = "A miniature avah filled with meat. This one has salsa on it."
	icon_state = "avah_full_salsa"
	bitten_state = "avah_half_salsa"
	bitesize = 2
	filling_color = "#FF4D36"

/obj/item/reagent_containers/food/snacks/chip/miniavah/guac
	name = "guac mini-avah"
	desc = "A miniature avah filled with meat. This one has guac on it."
	icon_state = "avah_full_guac"
	bitten_state = "avah_half_guac"
	bitesize = 2
	filling_color = "#35961D"

/obj/item/reagent_containers/food/snacks/chip/miniavah/sourcream
	name = "sourcream mini-avah"
	desc = "A miniature avah filled with meat. This one has sour cream on it."
	icon_state = "avah_full_sourcream"
	bitten_state = "avah_half_sourcream"
	bitesize = 2
	filling_color = "#e4e4e4"

/obj/item/reagent_containers/food/snacks/chip/miniavah/tajhummus
	name = "hummus mini-avah"
	desc = "A miniature avah filled with meat. This one has sweet hummus on it."
	icon_state = "avah_full_hummus"
	bitten_state = "avah_half_hummus"
	bitesize = 2
	filling_color = "#cca628"
```

File: icons/obj/food.dmi
[the remark this answers is not in the document]
Comment (Spookat) [reply icons/obj/food.dmi:1]: done, thank you

File: html/changelogs/ramke - tajfoodexpansion.yml (hunk 4 above)
Comment (Spookat) [reply html/changelogs/ramke - tajfoodexpansion.yml]: added references to some dishes in there.

File: html/changelogs/ramke - tajfoodexpansion.yml [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - rscadd: "Adds Tajara-themed food expansion, 17 new dishes. This includes variations of earthen-root dishes, hardbread pudding, sandwiches, dips, avahs and more."
```
Comment (SleepyGemmy) [inline html/changelogs/ramke - tajfoodexpansion.yml:41]:
```suggestion
  - rscadd: "Adds 17 new Tajara-themed foods. This includes variations of earthen-root dishes, hardbread pudding, sandwiches, dips, avahs, and more."
```

File: html/changelogs/ramke - tajfoodexpansion.yml [hunk 10]
```diff
@@ -0,0 +1,42 @@
...
+  - rscadd: "New garlic sauce sprite."
```
Comment (SleepyGemmy) [inline html/changelogs/ramke - tajfoodexpansion.yml:42]:
```suggestion
  - rscadd: "Adds 17 new Tajara-themed foods. This includes variations of earthen-root dishes, hardbread pudding, sandwiches, dips, avahs, and more."
  - imageadd: "New garlic sauce sprite."
```

**Review by SleepyGemmy — APPROVED** [review]

**Review by FluffyGhoster — APPROVED** [review]

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 11]
```diff
@@ -4334,6 +4334,22 @@
...
+	name = "hummus chip"
+	desc = "A portion sized chip good for dipping. This one has hummus on it."
+	icon_state = "chip_hummus"
+	bitten_state = "chip_half_hummus"
+	bitesize = 2
+	filling_color = "#cca628"
+
 /obj/item/reagent_containers/food/snacks/chip/nacho
 	name = "nacho chip"
 	desc = "A nacho ship stray from a plate of cheesy nachos."
```
Comment (Generalcamo) [inline code/modules/reagents/reagent_containers/food/snacks.dm:4355]:
```suggestion
	desc = "A stray nacho chip from a plate of cheesy nachos."
```
We're here anyway, might as well.

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 12]
```diff
@@ -4366,6 +4382,22 @@
 	bitesize = 2
 	filling_color = "#FFF454"
 
+/obj/item/reagent_containers/food/snacks/chip/nacho/sourcream
+	name = "nacho chip"
+	desc = "A nacho ship stray from a plate of cheesy nachos. This one has extra sour cream on it."
```
Comment (Generalcamo) [inline code/modules/reagents/reagent_containers/food/snacks.dm:4387]:
```suggestion
	desc = "A stray nacho chip from a plate of cheesy nachos. This one has extra sour cream on it."
```

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 13]
```diff
@@ -4366,6 +4382,22 @@
...
+	icon_state = "chip_nacho_sourcream"
+	bitten_state = "chip_half_sourcream"
+	bitesize = 2
+	filling_color = "#e4e4e4"
+
+/obj/item/reagent_containers/food/snacks/chip/nacho/tajhummus
+	name = "nacho chip"
+	desc = "A nacho ship stray from a plate of cheesy nachos. This one has extra hummus on it."
```
Comment (Generalcamo) [inline code/modules/reagents/reagent_containers/food/snacks.dm:4395]:
```suggestion
	desc = "A stray nacho chip from a plate of cheesy nachos. This one has extra hummus on it."
```

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 14]
```diff
@@ -4476,12 +4512,35 @@
...
+	avahtrans = /obj/item/reagent_containers/food/snacks/chip/miniavah/guac
 	chiptrans = /obj/item/reagent_containers/food/snacks/chip/guac
 	icon_state = "dip_guac"
 	reagents_to_add = list(/singleton/reagent/nutriment = 20)
 	reagent_data = list(/singleton/reagent/nutriment = list("guacmole" = 20))
 	filling_color = "#35961D"
 
+/obj/item/reagent_containers/food/snacks/dip/sarmikhir
+	name = "sarmikhir"
+	desc = "A traditional Tajaran cream made of fermented Fatshouter milk. Traditionally used with bread or as condiment."
```
Comment (Generalcamo) [inline code/modules/reagents/reagent_containers/food/snacks.dm:4524]:
```suggestion
	desc = "A traditional tajaran cream made of fermented Fatshouter milk. Traditionally used with bread or as condiment."
```

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 15]
```diff
@@ -4476,12 +4512,35 @@
...
+	avahtrans = /obj/item/reagent_containers/food/snacks/chip/miniavah/sourcream
+	chiptrans = /obj/item/reagent_containers/food/snacks/chip/sourcream
+	icon_state = "dip_sourcream"
+	reagents_to_add = list(/singleton/reagent/nutriment = 20)
+	reagent_data = list(/singleton/reagent/nutriment = list("sour cream" = 20))
+	filling_color = "#e4e4e4"
+
+/obj/item/reagent_containers/food/snacks/dip/tajhummus
+	name = "hrikhir"
+	desc = "A savoury Tajaran dip, typically paired with avahs."
```
Comment (Generalcamo) [inline code/modules/reagents/reagent_containers/food/snacks.dm:4535]:
```suggestion
	desc = "A savoury tajaran dip, typically paired with avahs."
```

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 16]
```diff
@@ -5336,6 +5397,14 @@
 	It is known for its hard crust, bland taste and for being long lasting. The hard bread was usually prepared for long journeys, hard winters or military campaigns, \
 	due to its shelf life. Certain folk stories and jokes claim that such food could also be used as an artillery ammunition or thrown at besieging armies during sieges."
 
+/obj/item/reagent_containers/food/snacks/hardbread_slice
+	name = "adhomian hard bread slice"
+	desc = "A long-lasting Tajaran bread slice. It is usually prepared for long journeys, hard winters, or military campaigns."
```
Comment (Generalcamo) [inline code/modules/reagents/reagent_containers/food/snacks.dm:5402]:
```suggestion
	desc = "A long-lasting tajaran bread slice. It is usually prepared for long journeys, hard winters, or military campaigns."
```
"Tajara" is not a proper noun, just like "human
 isn't.

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 17]
```diff
@@ -5419,6 +5488,199 @@
...
+	icon_state = "earthenroot_chopped"
+	trash = /obj/item/trash/plate
+	filling_color = "#5498b8"
+	reagents_to_add = list(/singleton/reagent/nutriment = 3)
+	reagent_data = list(/singleton/reagent/nutriment = list("sweet, chunky earthenroot" = 3))
+	bitesize = 2
+
+/obj/item/reagent_containers/food/snacks/earthenroot_fries
+	name = "earthen-root fries"
+	desc = "A plate full of fresh earthen-root fries. A crispy sweet Adhomian treat."
```
Comment (Generalcamo) [inline code/modules/reagents/reagent_containers/food/snacks.dm:5513]:
```suggestion
	desc = "A plate full of fresh earthen-root fries. A crispy and sweet Adhomian treat."
```

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 18]
```diff
@@ -5419,6 +5488,199 @@
...
+/obj/item/reagent_containers/food/snacks/sarmikhir_sandwich
+	name = "sarmikhir sandwich"
+	desc = "A Tajaran sandwich with hard bread and sour cream. A staple of the Hadiist breakfast menu."
+	icon_state = "hardbread_sourcream"
+	reagents_to_add = list(/singleton/reagent/nutriment = 12)
+	reagent_data = list(/singleton/reagent/nutriment = list("crusty bread" = 2, "sour cream" = 2))
+
+/obj/item/reagent_containers/food/snacks/soup/tajfish
+	name = "adhomian fish soup"
+	desc = "A creamy Adhomian fish soup garnished with sweet herbs."
```
Comment (Generalcamo) [inline code/modules/reagents/reagent_containers/food/snacks.dm:5575]:
```suggestion
	desc = "A creamy Adhomian fish soup, garnished with sweet herbs."
```

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 19]
```diff
@@ -5419,6 +5488,199 @@
...
+	desc = "An earthen-root stuffed with adhomian meat. Crunchy on the outside, savoury on the inside."
+	icon_state = "earthenroot_stuffed"
+	trash = /obj/item/trash/plate
+	reagents_to_add = list(/singleton/reagent/nutriment = 8, /singleton/reagent/nutriment/protein = 7)
+	reagent_data = list(/singleton/reagent/nutriment = list("sweet earthen-root" = 3), /singleton/reagent/nutriment/protein = list("smoky, salty meat"))
+	bitesize = 3
+
+/obj/item/reagent_containers/food/snacks/clam_pasta
+	name = "ras'val pasta"
+	desc = "A Tajaran pasta made from earthen-root, boiled with Ras'val clams. For the seafood lovers that can't handle spice."
```
Comment (Generalcamo) [inline code/modules/reagents/reagent_containers/food/snacks.dm:5551]:
```suggestion
	desc = "A tajaran pasta made from earthen-root, boiled with Ras'val clams. For the seafood lovers that can't handle spice."
```

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 20]
```diff
@@ -5419,6 +5488,199 @@
...
+	icon_state = "earthenroot_clambake"
+	trash = /obj/item/trash/plate
+	reagents_to_add = list(/singleton/reagent/nutriment = 6, /singleton/reagent/nutriment/protein/seafood = 6)
+	reagent_data = list(/singleton/reagent/nutriment = list("sweet and salty earthen-root" = 3), /singleton/reagent/nutriment/protein/seafood = list("salty clam meat"))
+	bitesize = 3
+
+/obj/item/reagent_containers/food/snacks/fermented_worm_sandwich
+	name = "fermented hma'trra sandwich"
+	desc = "A Tajaran delicacy that stinks as bad as it looks. Some claim the meat is an acquired taste and swear by its flavour."
```
Comment (Generalcamo) [inline code/modules/reagents/reagent_containers/food/snacks.dm:5560]:
```suggestion
	desc = "A tajaran delicacy that stinks as bad as it looks. Some claim the meat is an acquired taste and swear by its flavour."
```

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 21]
```diff
@@ -5419,6 +5488,199 @@
...
+	icon_state = "hardbread_fermented"
+	reagents_to_add = list(/singleton/reagent/nutriment = 10, /singleton/reagent/nutriment/protein/seafood = 15, /singleton/reagent/ammonia = 10)
+	reagent_data = list(/singleton/reagent/nutriment = list("crusty bread" = 2), /singleton/reagent/nutriment/protein/seafood = list("salty, tangy fish" = 3))
+	bitesize = 3
+
+/obj/item/reagent_containers/food/snacks/sarmikhir_sandwich
+	name = "sarmikhir sandwich"
+	desc = "A Tajaran sandwich with hard bread and sour cream. A staple of the Hadiist breakfast menu."
```
Comment (Generalcamo) [inline code/modules/reagents/reagent_containers/food/snacks.dm:5568]:
```suggestion
	desc = "A tajaran sandwich with hard bread and sour cream. A staple of the Hadiist breakfast menu."
```

**Review by Alberyk — APPROVED** [review]

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 22]
```diff
@@ -5419,6 +5488,199 @@
...
+	desc = "A Tajaran salad containing earthen-root, sarmikhir, herbs and cream. A great option for the more healthy-minded."
+	icon_state = "earthenroot_salad"
+	trash = /obj/item/trash/snack_bowl
+	reagents_to_add = list(/singleton/reagent/nutriment = 10)
+	reagent_data = list(/singleton/reagent/nutriment = list("sweet chunky earthen-root" = 2, "sour cream" = 2, "oily berries" = 2, "sweet herbs" = 2))
+	bitesize = 3
+
+/obj/item/reagent_containers/food/snacks/stuffed_earthenroot
+	name = "stuffed earthen-root"
+	desc = "An earthen-root stuffed with adhomian meat. Crunchy on the outside, savoury on the inside."
```
Comment (Generalcamo) [inline code/modules/reagents/reagent_containers/food/snacks.dm:5542]:
```suggestion
	desc = "An earthen-root stuffed with Adhomian meat. Crunchy on the outside, savoury on the inside."
```
Adhomai is a proper noun, and its denotation should follow suit.

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 23]
```diff
@@ -5419,6 +5488,199 @@
...
+
+/obj/item/reagent_containers/food/snacks/hardbread_bun
+	name = "hardbread bun"
+	desc = "A bun made out of hardbread pudding. Topped with sweet cream."
+	icon_state = "hardbread_pudding_bun"
+	trash = /obj/item/trash/snack_bowl
+
+/obj/item/reagent_containers/food/snacks/tunneler_meategg
+	name = "tunneler meategg"
+	desc = "An adhomian ice tunneler egg cooked with a layer of meat. All the protein that you'll need!"
```
Comment (Generalcamo) [inline code/modules/reagents/reagent_containers/food/snacks.dm:5599]:
```suggestion
	desc = "An Adhomian ice tunneler egg cooked with a layer of meat. All the protein that you'll need!"
```

File: html/changelogs/ramke - tajfoodexpansion.yml [hunk 24]
```diff
@@ -0,0 +1,42 @@
...
+# Optional: Remove this file after generating master changelog.  Useful for PR changelogs that won't get used again.
+delete-after: True
+
+# Any changes you've made.  See valid prefix list above.
+# INDENT WITH TWO SPACES.  NOT TABS.  SPACES.
+# SCREW THIS UP AND IT WON'T WORK.
+# Also, all entries are changed into a single [] after a master changelog generation. Just remove the brackets when you add new entries.
+# Please surround your changes in  double quotes ("), as certain characters otherwise screws up compiling. The quotes will not show up in the changelog.
+changes:
+  - rscadd: "Adds 17 new Tajara-themed foods. This includes variations of earthen-root dishes, hardbread pudding, sandwiches, dips, avah, and more."
```
Comment (Generalcamo) [inline html/changelogs/ramke - tajfoodexpansion.yml:41]:
```suggestion
  - rscadd: "Adds 17 new tajara-themed foods. This includes variations of earthen-root dishes, hardbread pudding, sandwiches, dips, avah, and more."
```

**Review by Generalcamo — CHANGES_REQUESTED** [review]

File: code/modules/reagents/reagent_containers/food/snacks.dm (hunk 14 above)
Comment (SleepyGemmy) [reply code/modules/reagents/reagent_containers/food/snacks.dm:4524]: wiki says otherwise.

Comment (Spookat) [reply code/modules/reagents/reagent_containers/food/snacks.dm:4524]: following wiki formatting, going to keep it capitalised

File: code/modules/reagents/reagent_containers/food/snacks.dm (hunk 23 above)
Comment (Spookat) [reply code/modules/reagents/reagent_containers/food/snacks.dm]: fixed

File: code/modules/reagents/reagent_containers/food/snacks.dm (hunk 22 above)
Comment (Spookat) [reply code/modules/reagents/reagent_containers/food/snacks.dm]: fixed

File: code/modules/reagents/reagent_containers/food/snacks.dm (hunk 17 above)
Comment (Spookat) [reply code/modules/reagents/reagent_containers/food/snacks.dm:5513]: crispy sweet is a valid descriptor

**Pull request merged**